### PR TITLE
WIP: Save to PDF 

### PR DESCRIPTION
**Pull request by grero**
My quick-and-dirty fix for saving to PDF. I'm sure lots can be improved here : )

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,5 +1,5 @@
 julia 0.7
-
+Gtk
```
[the remark this answers is not in the document]
Comment (grero) [reply REQUIRE]: You're absolutely right, I think I put it there after getting an error in a previous version of the code.

**Comment by asinghvi17** [comment]
@grero does this work / should it be merged?

**Comment by asinghvi17** [comment]
bump

**Comment by grero** [comment]
Sorry, I haven't really had to time to work on this. It worked for a basic plot a while back, but after merging in the latest master, I got an error when trying to save the plot:

```julia
ERROR: MethodError: no method matching push_screen!(::Scene, ::Tuple{CairoMakie.CairoBackend,Scene})
```
I'm not really sure what that function does, but I'm guessing it's something that needs to be added to CairoMakie to align with upstream changes in AbstractPlotting?

**Comment by SimonDanisch** [comment]
Maybe merge in master?

**Comment by asinghvi17** [comment]
I'll open a rebased PR.

**Comment by asinghvi17** [comment]
Superseded by #31, thanks @grero!

**Pull request closed without merging**